File: app/src/main/java/org/jellyfin/androidtv/data/compat/VideoOptions.java
package org.jellyfin.androidtv.data.compat;

public class VideoOptions extends AudioOptions {
    private Integer AudioStreamIndex;
    private Integer SubtitleStreamIndex;

    public final Integer getAudioStreamIndex() {
        return AudioStreamIndex;
    }

    public final void setAudioStreamIndex(Integer value) {
        AudioStreamIndex = value;
    }

    public final Integer getSubtitleStreamIndex() {
        return SubtitleStreamIndex;
    }

    public final void setSubtitleStreamIndex(Integer value) {
        SubtitleStreamIndex = value;
    }
}

File: app/src/main/java/org/jellyfin/androidtv/data/compat/VideoOptions.kt
package org.jellyfin.androidtv.data.compat

class VideoOptions : AudioOptions() {
	var audioStreamIndex: Int? = null
	var subtitleStreamIndex: Int? = null

	/**
	 * When true, the server may copy the video stream into the output container (when compatible).
	 * When false, the server should transcode the video stream.
	 */
	var allowVideoStreamCopy: Boolean = true
}
